doc/api/replication.md

```diff
@@ -0,0 +1,106 @@
+# Thali Cordova Peer Communication API #
+
+## Methods:
+- `StartBroadcasting`
+- `StopBroadcasting`
+- `Connect`
+- `Disconnect`
+- `KillConnection`
+
+Events:
+- `peerAvailabilityChanged`
+- `networkChanged`
+
+***
+
+METHODS:
+
+***
+
+### `StartBroadcasting(deviceName, portNumber, callback)`
+
+This method starts advertising or broadcasting its availability at the following port that PouchDB/Express-PouchDB is listening.  
+
```

Comment:
What happens if someone calls StartBroadcasting twice with the same deviceName and portNumber without calling StopBroadcasting in between?
What happens if someone calls StartBroadcasting twice with different deviceNames and portNumbers without calling StopBroadcasting in between?
In general I believe the right answer is that we only allow a single outstanding call to StartBroadcasting regardless of the arguments and a second call without a StopBroadcasting in between is an error.